Sources/VaporToolbox/TemplateRenderer.swift

```diff
@@ -0,0 +1,226 @@
+            }
+            return input.hasPrefix("y")
+        }
+    }
+
+    /// Renders a Mustache file from the template using the provided context.
+    ///
+    /// - Parameters:
+    ///   - file: The file to render.
+    ///   - sourceURL: The URL of the template source.
+    ///   - destinationURL: The URL of the destination folder where the project will be generated.
+    ///   - context: The context to use to render the files.
+    private func render(
+        _ file: TemplateManifest.File,
+        from sourceURL: URL,
+        to destinationURL: URL,
+        with context: [String: Any]
+    ) throws {
+        // If the file has a condition on whether it should be rendered or not
+        // check for the condition in the context, including nested variables.
+        if case .exists(let variable) = file.condition {
+            let components = variable.split(separator: ".").map { String($0) }
+            var subContext: Any = context
+
+            for component in components {
+                guard
+                    let dict = subContext as? [String: Any],
+                    let value = dict[component]
+                else {
+                    return
+                }
+
+                if let bool = value as? Bool, !bool {
+                    return
+                }
+
+                subContext = value
+            }
+        }
+
+        let destinationFileName =
+            if let dynamicName = file.dynamicName {
+                try MustacheTemplate(string: dynamicName).render(context)
+            } else {
+                file.name
+            }
+        let destinationFileURL = destinationURL.appending(path: destinationFileName)
+
+        switch file.type {
+        case .file(let dynamic):
+            if dynamic {
+                let template = try String(contentsOf: sourceURL.appending(path: file.name), encoding: .utf8)
+                try MustacheTemplate(string: template).render(context)
+                    .write(to: destinationFileURL, atomically: true, encoding: .utf8)
+            } else {
+                try FileManager.default.moveItem(at: sourceURL.appending(path: file.name), to: destinationFileURL)
+            }
+            if self.verbose { print("+ " + file.name) }
+        case .folder(let files):
+            let folder = file
+            try FileManager.default.createDirectory(at: destinationFileURL, withIntermediateDirectories: false)
+            for file in files {
+                try self.render(
+                    file,
+                    from: sourceURL.appending(path: folder.name, directoryHint: .isDirectory),
+                    to: destinationFileURL,
+                    with: context
+                )
+            }
+        }
+    }
+}
+
+extension StringProtocol {
+    var kebabcased: String {
+        self
+            .components(separatedBy: .alphanumerics.inverted)
+            .filter { !$0.isEmpty }
```

Comment:
```suggestion
            .split(whereSeparator: { !$0.isLetter })
```
(The filter is not needed; `omittingEmptySubsequences` defaults to `true` in this API.)